### PR TITLE
Disable JS

### PR DESCRIPTION
**Pull request by ilya-lavrenov**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by ilya-lavrenov** [comment]
@conda-forge-admin, please rerender

File: .ci_support/linux_64_.yaml [hunk 1]
```diff
@@ -19,7 +19,7 @@ cxx_compiler_version:
 docker_image:
 - quay.io/condaforge/linux-anvil-cos7-x86_64
 flatbuffers:
-- 23.5.26
+- 24.3.6
```
Comment (ilya-lavrenov) [inline .ci_support/linux_64_.yaml:22]:
```suggestion
- 23.5.26
```

File: .ci_support/osx_64_.yaml [hunk 2]
```diff
@@ -19,7 +19,7 @@ cxx_compiler:
 cxx_compiler_version:
 - '16'
 flatbuffers:
-- 23.5.26
+- 24.3.6
```
Comment (ilya-lavrenov) [inline .ci_support/osx_64_.yaml:22]:
```suggestion
- 23.5.26
```

File: .ci_support/win_64_python3.10.____cpython.yaml [hunk 3]
```diff
@@ -9,7 +9,7 @@ channel_targets:
 cxx_compiler:
 - vs2019
 flatbuffers:
-- 23.5.26
+- 24.3.6
```
Comment (ilya-lavrenov) [inline .ci_support/win_64_python3.10.____cpython.yaml:12]:
```suggestion
- 23.5.26
```

File: .ci_support/win_64_python3.11.____cpython.yaml [hunk 4]
```diff
@@ -9,7 +9,7 @@ channel_targets:
 cxx_compiler:
 - vs2019
 flatbuffers:
-- 23.5.26
+- 24.3.6
```
Comment (ilya-lavrenov) [inline .ci_support/win_64_python3.11.____cpython.yaml:12]:
```suggestion
- 23.5.26
```

File: .ci_support/win_64_python3.12.____cpython.yaml [hunk 5]
```diff
@@ -9,7 +9,7 @@ channel_targets:
 cxx_compiler:
 - vs2019
 flatbuffers:
-- 23.5.26
+- 24.3.6
```
Comment (ilya-lavrenov) [inline .ci_support/win_64_python3.12.____cpython.yaml:12]:
```suggestion
- 23.5.26
```

File: .ci_support/win_64_python3.8.____cpython.yaml [hunk 6]
```diff
@@ -9,7 +9,7 @@ channel_targets:
 cxx_compiler:
 - vs2019
 flatbuffers:
-- 23.5.26
+- 24.3.6
```
Comment (ilya-lavrenov) [inline .ci_support/win_64_python3.8.____cpython.yaml:12]:
```suggestion
- 23.5.26
```

File: .ci_support/win_64_python3.9.____cpython.yaml [hunk 7]
```diff
@@ -9,7 +9,7 @@ channel_targets:
 cxx_compiler:
 - vs2019
 flatbuffers:
-- 23.5.26
+- 24.3.6
```
Comment (ilya-lavrenov) [inline .ci_support/win_64_python3.9.____cpython.yaml:12]:
```suggestion
- 23.5.26
```

File: .ci_support/osx_arm64_.yaml [hunk 8]
```diff
@@ -17,7 +17,7 @@ cxx_compiler:
 cxx_compiler_version:
 - '16'
 flatbuffers:
-- 23.5.26
+- 24.3.6
```
Comment (ilya-lavrenov) [inline .ci_support/osx_arm64_.yaml:20]:
```suggestion
- 23.5.26
```

File: .ci_support/linux_aarch64_.yaml [hunk 9]
```diff
@@ -23,7 +23,7 @@ cxx_compiler_version:
 docker_image:
 - quay.io/condaforge/linux-anvil-cos7-x86_64
 flatbuffers:
-- 23.5.26
+- 24.3.6
```
Comment (ilya-lavrenov) [inline .ci_support/linux_aarch64_.yaml:26]:
```suggestion
- 23.5.26
```

File: .ci_support/osx_64_.yaml [hunk 10]
```diff
@@ -19,7 +19,7 @@ cxx_compiler:
 cxx_compiler_version:
 - '16'
 flatbuffers:
-- 24.3.6
+- 23.5.26
```
Comment (hmaarrfk) [inline .ci_support/osx_64_.yaml:22]: I take it things are not compatible with flatbuffers 24.3?

Comment (ilya-lavrenov) [reply .ci_support/osx_64_.yaml:22]: such version is not available
image

Comment (hmaarrfk) [reply .ci_support/osx_64_.yaml:22]: lets see... https://github.com/conda-forge/flatbuffers-feedstock/issues/50

**Pull request merged**